### PR TITLE
Core styles for demo store

### PR DESCRIPTION
**Pull request by artursmet**
Re-opened pull request from main saleor repository.

### CODE REVIEW

File: saleor/static/js/demo/storefront.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  $addressForm.slideDown('slow')
+})
+$addressHide.click((e) => {
+  $addressForm.slideUp('slow')
+})
+if ($initialValue) {
+  $addressForm.slideDown(0)
+} else {
+  $addressForm.slideUp(0)
+}
```
Comment (benekex2) [inline saleor/static/js/demo/storefront.js:57]: missing new line

File: saleor/static/scss/demo/components/_breadcrumbs.scss [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    &:not(:last-of-type):after {
+      content: '/';
+      display: inline-block;
+      margin-left: $global-margin / 3;
+    }
+    a {
+      color: $body-color;
+    }
+  }
+}
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_breadcrumbs.scss:14]: missing new line

File: saleor/static/scss/demo/components/_buttons.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    background-color: $gray;
+    border: 0;
+    padding: $global-padding / 2 $global-padding * 2;
+  }
+  &.btn-link {
+    padding: 0;
+    vertical-align: baseline;
+    color: $gray;
+  }
+}
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_buttons.scss:53]: missing new line in all new scss files

File: saleor/static/scss/demo/components/_cart-dropdown.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
+.cart-dropdown {
+  position: absolute;
+  width: 460px;
+  top: 0;
+  padding-top: 50px;
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_cart-dropdown.scss]: Use $global-padding

File: saleor/static/scss/demo/components/_footer.scss [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      margin-right: $global-margin;
+    }
+  }
+}
+
+//make footer stick to the bottom
+.maincontent {
+  min-height: calc(100vh - 294px - 191px - 30px);
+  @include media-breakpoint-down(sm) {
+    margin-top: 100px;
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_footer.scss]: Use &global-margin

File: saleor/static/scss/demo/components/_header.scss [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      &:hover .cart-dropdown {
+        display: block;
+      }
+      @include media-breakpoint-up(md) {
+        margin-top: $global-margin / 2;
+      }
+      &__icon {
+        margin-left: $global-margin;
+      }
+      margin-right: 15px;
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_header.scss]: Use &global-margin

File: saleor/static/scss/demo/components/_cart-dropdown.scss [hunk 7]
```diff
@@ -0,0 +1,50 @@
+.cart-dropdown {
+  position: absolute;
+  width: 460px;
```
Comment (benekex2) [inline saleor/static/scss/demo/components/_cart-dropdown.scss]: we shouldn't use rem instead pixels ?

File: webpack.config.js [hunk 8]
```diff
@@ -20,16 +20,18 @@ var providePlugin = new webpack.ProvidePlugin({
   jQuery: 'jquery',
   'window.jQuery': 'jquery',
   Backbone: 'backbone',
-  'window.Backbone': 'backbone'
+  'window.Backbone': 'backbone',
```
Comment (benekex2) [inline webpack.config.js]: We use Backbone on this project?

**Review by benekex2 — CHANGES_REQUESTED** [review]

File: saleor/static/scss/demo/storefront.scss [hunk 9]
```diff
@@ -13,4 +13,5 @@
 @import "components/login";
 @import "components/product-list";
 @import "components/cart-dropdown";
-@import "components/checkout";
\ No newline at end of file
+@import "components/checkout";
+@import "components/forms";
```
Comment (car3oon) [inline saleor/static/scss/demo/storefront.scss]: new line missing

File: saleor/static/scss/demo/components/_footer.scss [hunk 10]
```diff
@@ -31,4 +31,7 @@
 //make footer stick to the bottom
 .maincontent {
   min-height: calc(100vh - 294px - 191px - 30px);
+  @include media-breakpoint-down(sm) {
+    margin-top: 85px;
+  }
 }
```
Comment (car3oon) [inline saleor/static/scss/demo/components/_footer.scss]: new line missing

File: saleor/static/scss/demo/components/_forms.scss [hunk 11]
```diff
@@ -0,0 +1,6 @@
+.form-control {
+  color: $body-color;
+  &:focus {
+    color: $body-color;
+  }
+}
```
Comment (car3oon) [inline saleor/static/scss/demo/components/_forms.scss:6]: new line missing

File: templates/demo/cart-dropdown.html [hunk 12]
```diff
@@ -0,0 +1,51 @@
+{% load i18n %}
+{% load staticfiles %}
+{% load gross from prices_i18n %}
+
+<div class="cart-dropdown hidden-md-down">
+  <div class="container">
+    <!--
```
Comment (car3oon) [inline templates/demo/cart-dropdown.html]: delete unused code

File: saleor/static/js/demo/storefront.js [hunk 13]
```diff
@@ -54,4 +54,41 @@ if ($initialValue) {
...
+          var newQunatity = $('.cart-dropdown__total').data('quantity')
+          $('.badge').html(newQunatity).removeClass('hidden-xs-up')
+          $cartDropdown.addClass("show")
+          setTimeout((e) => {
+            $cartDropdown.removeClass('show');
+          }, 2000)
+      })
+    }
+  })
+});
```
Comment (car3oon) [inline saleor/static/js/demo/storefront.js]: new line missing

File: saleor/static/scss/demo/components/_cart-dropdown.scss [hunk 14]
```diff
@@ -47,4 +49,7 @@
       margin: $global-margin 0;
     }
   }
+  &.show {
+    display: block;
+  }
 }
```
Comment (car3oon) [inline saleor/static/scss/demo/components/_cart-dropdown.scss]: new line missing

File: saleor/static/js/demo/storefront.js [hunk 15]
```diff
@@ -54,4 +54,41 @@ if ($initialValue) {
...
+    data: {
+      variant: variant, 
+      quantity: quantity
+    },
+    success: function() {
+      $.get(summaryLink, (data) => {
+          $cartDropdown.html(data)
+          var newQunatity = $('.cart-dropdown__total').data('quantity')
+          $('.badge').html(newQunatity).removeClass('hidden-xs-up')
+          $cartDropdown.addClass("show")
```
Comment (car3oon) [inline saleor/static/js/demo/storefront.js:87]: Check whole file for semicolons inconsistency

**Comment by artursmet** [comment]
Is it ready to merge? @benekex2 @car3oon?

**Pull request merged**